packages/collector/test/tracing/cloud/azure/blob/app.js

```diff
@@ -0,0 +1,326 @@
+/*
+ * (c) Copyright IBM Corp. 2023
+ */
+
+/* eslint-disable no-console */
+
+'use strict';
+
+require('../../../../..')();
+
+const { BlobServiceClient, BlobBatchClient, BlobClient, StorageSharedKeyCredential } = require('@azure/storage-blob');
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const port = require('../../../../test_util/app-port')();
+const agentPort = process.env.INSTANA_AGENT_PORT;
+const app = express();
+const logPrefix = `Express / azure blob App (${process.pid}):\t`;
+const fs = require('fs');
+const request = require('request-promise-native');
```

Comment:
please use `node-fetch` instead of `request-promise-native`.

Reply:
changed